terraform/README.md

```diff
@@ -0,0 +1,65 @@
+# Terraform module for `cos-registration-server`
+
+This is a Terraform module facilitating the deployment of the `cos-registration-server` charm,
+using the [Terraform Juju provider](https://github.com/juju/terraform-provider-juju/).
+For more information,
+refer to the provider [documentation](https://registry.terraform.io/providers/juju/juju/latest/docs).
+
+## Requirements
+
+This module requires a Juju K8s model to be available.
+Refer to the [usage section](#usage) below for more details.
+
+## Usage
+
+Users should ensure that Terraform is aware of the `juju_model` dependency of the charm module.
```

Comment:
Does this mean: Users should ensure that Terraform is aware of the `juju_model` the charm should be deployed to? why dependency

Reply:
Yeah, this essentially states that this module does not create the model. The model creation and management is left to the module caller.